### PR TITLE
Added oscsendfile target to CMake build

### PR DESCRIPTION
**Pull request by gaborpapp**

### CODE REVIEW

File: cmake/CMakeLists.txt [hunk 1]
```diff
@@ -163,7 +165,7 @@ endif()
 set(BUILD_LANGUAGE C CACHE STRING "Build language (C or CXX)")
 mark_as_advanced(BUILD_LANGUAGE)
 set_source_files_properties(
-    ${LIBRARY_SOURCES} ${OSCDUMP_SOURCES} ${OSCSEND_SOURCES}
+	${LIBRARY_SOURCES} ${OSCDUMP_SOURCES} ${OSCSEND_SOURCES} ${OSCSENDFILE_SOURCES}
```
Comment (h3xx) [inline cmake/CMakeLists.txt:168]: Spaces replaced by tabs. You should probably keep tabs-vs-spaces consistent.

File: cmake/CMakeLists.txt [hunk 2]
```diff
@@ -371,7 +374,7 @@ configure_file(
 add_definitions(-DHAVE_CONFIG_H)
 
 install(
-    TARGETS ${LIBRARY_STATIC} ${LIBRARY_SHARED} ${OSCDUMP} ${OSCSEND}
+	TARGETS ${LIBRARY_STATIC} ${LIBRARY_SHARED} ${OSCDUMP} ${OSCSEND} ${OSCSENDFILE}
```
Comment (h3xx) [inline cmake/CMakeLists.txt:377]: Spaces replaced by tabs. You should probably keep tabs-vs-spaces consistent.

File: cmake/CMakeLists.txt (hunk 1 above)
Comment (h3xx) [inline cmake/CMakeLists.txt:168]: Spaces replaced by tabs.

File: cmake/CMakeLists.txt (hunk 2 above)
Comment (gaborpapp) [reply cmake/CMakeLists.txt]: Thanks for the review. Pushed a commit to fix these issues.

**Review by h3xx — APPROVED** [review]

**Pull request merged**